index.bs

```diff
@@ -5822,6 +5822,12 @@ Since certificates are sizable relative to the storage capabilities of typical a
 
 Note: In order to interoperate, user agents storing large blobs on authenticators using [[!FIDO-CTAP]] are expected to use the provisions detailed in that specification for storing [=large, per-credential blobs=].
 
+Note: [[!FIDO-CTAP]] authenticators only support this extension for [=discoverable credentials=],
+and might return an error unless
+<code>{{PublicKeyCredentialCreationOptions/authenticatorSelection}}.{{AuthenticatorSelectionCriteria/residentKey}}</code>
+is set to {{ResidentKeyRequirement/preferred}} or {{ResidentKeyRequirement/required}}.
+This is not a general requirement for all kinds of authenticators.
```

Comment:
This final sentence is vague to me. I do not understand what "this" refers to, and what "all kinds" implies, e.g. which authenticator characteristics ?

Reply:
Would this make it clearer?
```suggestion
Authentictors other than [[!FIDO-CTAP]] ones do not in general restrict this extension to [=discoverable credentials=].
```

Reply:
sort of, but now that I understand I have comments on the entire paragraph ;-)

Reply:
saying "[fido-ctap] authenticators" would be introducing a new term not used elsewhere in the spec. those are "roaming authnrs".

```suggestion
Note: [=Roaming authenticators=], which use [[!FIDO-CTAP]] as their cross-platform transport protocol, only support this [=largeblob/Large Blob=] extension for [=discoverable credentials=],
and might return an error unless
<code>{{PublicKeyCredentialCreationOptions/authenticatorSelection}}.{{AuthenticatorSelectionCriteria/residentKey}}</code>
is set to {{ResidentKeyRequirement/preferred}} or {{ResidentKeyRequirement/required}}.
However, [=platform authenticators=] (which do not generally utilize [[!FIDO-CTAP]] internally) typically do not restrict this extension to [=discoverable credentials=].
```

Reply:
Continued here: https://github.com/w3c/webauthn/pull/1622#discussion_r669212648